### ISSUE TITLE
Ускорение

### ISSUE THREAD

**Issue by Alphabet82** [Anuken/Mindustry-Suggestions#2046]
Идея такова есть кнопка с выбором:2x, 5x и может 10х если надо. После выбора скорости все карта ускоряется:Время до следующей волны, скорость добычи и передвижение по ленте, скорость постройки чего либо. Надеюсь понятно объяснил

**Comment by BlueWolf3682** [comment]
sk7725/timecontrol

**Comment by Alphabet82** [comment]
Yes

**Comment by Qvertyes** [comment]
И в честь чего такие бусты?

**Comment by Alphabet82** [comment]
Это просто удобно

**Comment by Yell245** [comment]
А вот если бы ты проверил список того, что нельзя предлагать, то ты бы узнал, что это предложение там находится

**Comment by itcannotbe** [comment]
> The idea is there is a button with a choice: 2x, 5x and maybe 10x if necessary.  After choosing the speed, the whole map is accelerated: The time until the next wave, the speed of production and movement along the tape, the speed of building something.  I hope I explained clearly

No. Try reading the `README.md` to make sure your idea is not listed under the "A few things you shouldn't suggest" category.

And yes, the template should.have prevent someone from even suggesting this.

------

Нет. Попробуйте прочитать `README.md`, чтобы убедиться, что ваша идея не указана в категории« Несколько вещей, которые вам не следует предлагать ».

  И да, шаблон должен был помешать кому-либо даже предложить это
